### ISSUE TITLE
Accessibility: Language select field missing label

### ISSUE THREAD

**Issue by nicolaisweb** [globaleaks/GlobaLeaks#3574]
### What version of GlobaLeaks are you using?

4.12.8

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### Describe the issue

The `<select>` field for language selction doesn't have a `<label>` tag

### Proposed solution

Provide a `<label>` tag referring to the field's ID

**Comment by evilaliv3** [comment]
Closing as implemented
